common/beerocks/bwl/include/bwl/base_wlan_hal_types.h

```diff
@@ -46,17 +46,21 @@ enum class AntMode { Invalid = 0, ANT_1X1, ANT_2X2, ANT_3X3, ANT_4X4 };
 enum class WiFiChanBW { Invalid = 0, BW_20 = 20, BW_40 = 40, BW_80 = 80 };
 
 struct VAPElement {
+    /**
+     * Basic Service Set (i.e.: VAP name, e.g.: wlan0.0, wlan0.1, wlan0.2, ...).
+     */
+    std::string bss;
     std::string ssid;
     std::string mac;
     bool fronthaul;
     bool backhaul;
 
-    virtual bool operator==(const VAPElement &other) const
+    bool operator==(const VAPElement &other) const
```

Comment:
Since you visited this code and removed the `virtual`:
1. did you verify that there are no derived classes from this `struct`?
2. I think that the comparison itself should be visited as well. It looks like a mistake to have two `VAPElement` objects with the same `ssid` and `mac` but with different `fronthaul` or different `vap_name`.

Reply:
Actually for 1. if there is any derived class overriding this method, it would hopefully use the "override" specifier so the compiler would tell us that it no longer overrides anything.
In practice there are still some places where the specifier is missing so you're right, we have to check (I did, there are none).

Reply:
I removed `virtual` because there are no classes deriving from `VAPElement` and there's no virtual destructor (my editor was warning me about this).